requirements.txt

```diff
@@ -5,8 +5,6 @@ jsonargparse[signatures]  # CLI
 # other optional dependencies are
 # sentencepiece  # pythia, falcon, redpajama
 # tokenizers  #  llama-based models
-# bitsandbytes>=0.41.1  # quantize/bnb.py
```

Comment:
This is still an optional dependency that you need for quantization, so shouldn't it stay?

Reply:
note that this is commented out, the tutorials for quantization still mention that it should be installed.

The pin is now decided by lightning, not by lit-gpt. This is why I removed it